### ISSUE TITLE
Tabout doesn't work at all

### ISSUE THREAD

**Issue by KJ3RO** [abecodes/tabout.nvim#54]
The tabout extension doesn't seem to be working at all...

https://github.com/abecodes/tabout.nvim/assets/115525965/e961540e-20e4-49a5-99d2-915702cfe2ba

I included tabout in my plugins-setup.lua:
```
use({
  "abecodes/tabout.nvim",
  branch = "feature/tabout-md",
  wants = { "nvim-treesitter" },
  after = { "nvim-cmp" },
})
```
I also made a config file (copied from the docs):
```
local setup, tabout = pcall(require, "tabout")
if not setup then
  return
end

tabout.setup({
  tabkey = "<Tab>",
  backwards_tabkey = "<S-Tab>",
  act_as_tab = true,
  act_as_shift_tab = true,
  default_tab = "<C-t>",
  default_shift_tab = "<C-d>",
  enable_backwards = true,
  completion = true,
  tabouts = {
    { open = "'", close = "'" },
    { open = '"', close = '"' },
    { open = "`", close = "`" },
    { open = "(", close = ")" },
    { open = "[", close = "]" },
    { open = "{", close = "}" },
  },
  ignore_beginning = true, --[[ if the cursor is at the beginning of a filled element it will rather tab out than shift the content ]]
  exclude = {},
})
```

But it doesn't seem to work on any of the files. I did include it in my init.lua file as well (require("doyeon.plugins.tabout")) and I checked multiple issues and discussions by other and tried their solutions. None of them seemed to work. Am I doing something wrong?

**Comment by KJ3RO** [comment]
> How was this solved? 😕

The reason why this error occured was not because of the configuration of tabout but because it overlapped with treesitter and other plugins. I solved this by reinstalling plugins like treesitter. However, to completely solve this problem, I suggest you just start over by moving your neovim config to another file and installing plugins one by one to see which ones caused the problems and then edit your configurations for each plugin. I realize that this might not be the most efficent way but this is how I solved my problem.
